boards/shields/arduino_uno_click/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _arduino_uno_click:
+
+Arduino UNO click shield
+########################
+
+Overview
+********
+
+The Arduino UNO click is an extension to the Arduino UNO R3 headers.
+It's a simple shield that converts Arduino UNO R3 headers to two mikroBUS
+host sockets that allow you to connect many other click shields to your
+board.
+In other words, he Arduino UNO click will generally be used by other
+shields using the mikroBUS interface.
+
+Two mikroBUS headers are exposed by the overlay: mikrobus_headers_1 and
+mikrobus_headers_2, each corresponding to a socket on the Arduino UNO
+click shield.
+
+More information about the shield can be found at
+`Arduino UNO click shield website`_.
+
+Requirements
+************
+
+This shield can only be used with a board which provides a configuration
+for Arduino R3 connector.
+
+Since the Arduino UNO click has two sockets, the user must create a node
+alias depeding on which of the sockets is used:
+
+mikrobus_header: &mikrobus_header_1 {}; or
+mikrobus_header: &mikrobus_header_2 {};
```

Comment:
One of them should be &mikrobus_header, another probably &mikrobus_header_alt. Or add

`mikrobus_header: &mikrobus_header_1 {};`

to boards/shields/arduino_uno_click/arduino_uno_click.overlay

Reply:
Wouldn't adding 
mikrobus_header: &mikrobus_header_1 {}; 
to the end of overlay force all the connecting shields to use socket_1?

The way it's implemented now, both sockets are available, and it's up to the board overlay to decide which socket is used.

Reply:
> to the end of overlay force all the connecting shields to use socket_1?

Yes, that is the point. Only one of them can be used in the tree out of the box, `git grep mikrobus_header`, and that is `mikrobus_header`.

>The way it's implemented now, both sockets are available, and it's up to the board overlay to decide which socket is used.

Actually no, I do not see a generic way to choose one of them for a arbitrary shield at build time.

Reply:
The node label mikrobus_header could be set in the board overlay to decide which socket is used. 
But I don't have a strong preference about this. I used mikrobus_header_1 as default and assigned mikrobus_header label.

Reply:
> The node label mikrobus_header could be set in the board overlay to decide which socket is used.

It is about existing shields. But have an overlay per board is nonsense as well.